src/pages/components/notification/usage.mdx

```diff
@@ -302,57 +292,113 @@ dismissed.
-users who need more time reading the notification or who may want to refer back
-to the notification. All toast notifications cover content on the screen so they
-should always be easily dismissed.
+Toast notifications persist by default. Toasts do have the option to timeout and
+dismiss automatically after five seconds on the screen. They can also include a
+close button so users can dismiss them sooner. Toasts cover content on the
+screen so they should always be easily dismissed. Because toast notifications
+can dismiss automatically, users should be able to access them elsewhere after
+the toast disappears. This allows them to be accessible for users who need more
+time reading the notification or who may want to refer back to the notification.
 
-## Modifiers
+## Actionable notifications
 
-### High and low contrast
+Actionable notifications allow for interactive elements within a notification
+styled like an inline or toast notification. Actionable notifications, since
+they require user interaction, take focus when triggered and can be highly
+disruptive to screen readers and keyboard users. With actionable notifications,
+only one action is allowed per notification. This action frequently takes users
+to a flow or page related to the message, where they can resolve the
+notification. Consider using a notification center where a user can revisit and
+act on past notifications.
 
-Carbon supports high and low contrast style notifications. High-contrast
-notifications are best for critical messaging while low-contrast notifications
-are less visually disruptive for users.
+### Formatting
 
-It's up to the product team to decide which notification style to use in their
-product. Inline and toast notifications can use different styles but you should
-never mix styles within each notification variant. When in doubt, use
-low-contrast notifications.
+#### Inline
+
+Actionable inline notifications have a ghost button action that is adjacent to
+the title and body content. On mobile screens the action button wraps under the
+body content. This button should allow users to take further action on the
+notification.
+
+#### Toast
+
+Actionable toast notifications can include a button at the end of their body
+content. This button should be short and navigate users to a page or modal where
+they can take action to address the notification or find further information.
+Because toast notifications automatically dismiss, it is important that there
+are alternative routes to navigate to the link destination.
 
 <Row>
 <Column colLg={8}>
 
-![High and low contrast inline notifications](images/06_notification_contrast_608.png)
+![Example of actionable notifications](images/07_actionable_types_608.png)
 
 <Caption>
-    High contrast notification (top); low contrast notification (bottom)
+  Actionable toast notification (top); actionable inline notification (bottom)
 </Caption>
+
 </Column>
 </Row>
 
-### Actionable
+### Behavior
```

Comment:
I noticed there is a main H2 "Behaviors" section as well as H3 "Behavior" sections for the different variants. Maybe we just omit the H2 "Behaviors" and just let the H4 "Dismissals" be the only heading? Or we could rename the H2 to "Universal behaviors"?